configs/webpack.config.common.js

```diff
@@ -11,7 +12,7 @@ if(indexOf > -1 && args.length > indexOf+1){
 
 const newPath = path.join(__dirname, "../../../../", pathToJoin);
 
-const pkg = require(path.join(newPath, "package.json"));
+const pkg = injectLocalConfigs(require(path.join(newPath, "package.json")),newPath);
```

Comment:
Feature request, we should in webpack relative paths based on the project, maybe we can export them.